### PR TITLE
Remove ide

### PR DESCRIPTION
**Pull request by jiwon-lieb**
## Proposed changes

I removed all mentions of Klaytn IDE and replaced it with Remix Online IDE

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

File: docs/smart-contract/sample-contracts/erc-20/2-erc20.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+After deploying, you can invoke `balanceOf` with your account, which was used to deploy the contract. You will find there are `10000000000000` tokens available in your account as below. Because you set `decimal` as `8` when deploying the contract above, it minted a fixed number of `100000` tokens in the constructor, with one token having a decimal value of `10^8`. `totalSupply` method will return the total supply of tokens minted which should be also `10000000000000`.
 
-![ERC20-2-owner-token](./images/erc20-2-owner_token.png)
+![ERC20-2-owner-token](images/erc20-2-owner\_token.png)
 
-Now `MyERC20` is live !
+`MyERC20` is now live !
 
-## 2.2 Deploying smart contract using truffle <a id="2-2-deploying-smart-contract-using-truffle"></a>
+## 2.2 Deploying smart contract using truffle <a href="#2-2-deploying-smart-contract-using-truffle" id="2-2-deploying-smart-contract-using-truffle"></a>
```
Comment (kjhman21) [inline docs/smart-contract/sample-contracts/erc-20/2-erc20.md:21]:
```suggestion
## 2.2 Deploying smart contract using truffle <a id="2-2-deploying-smart-contract-using-truffle"></a>
```

File: docs/smart-contract/sample-contracts/erc-721/2-erc721.md [hunk 2]
```diff
@@ -1,36 +1,36 @@
-# 2. Deploying Smart Contract <a id="2-deploying-smart-contract"></a>
+# 2. Deploying Smart Contract
 
-You can use Klaytn IDE or use truffle to deploy above `MyERC721Card` smart contract.
+You can use Remix Online IDE or use truffle to deploy above `MyERC721Card` smart contract.
 
-## 2.1 Deploying smart contract using Klaytn IDE <a id="2-1-deploying-smart-contract-using-klaytn-ide"></a>
+## 2.1 Deploying smart contract using Remix Online IDE <a href="#2-1-deploying-smart-contract-using-klaytn-ide" id="2-1-deploying-smart-contract-using-klaytn-ide"></a>
```
Comment (kjhman21) [inline docs/smart-contract/sample-contracts/erc-721/2-erc721.md:5]:
```suggestion
## 2.1 Deploying smart contract using Remix Online IDE <a id="2-1-deploying-smart-contract-using-klaytn-ide"></a>
```

File: docs/getting-started/development-environment.md [hunk 3]
```diff
@@ -7,24 +7,21 @@
...
-* [Endpoint Node](../node/endpoint-node/README.md) is needed to connect to the Klaytn network and to issue an API call or send a transaction. 
-* `ken` is a Klaytn Endpoint Node binary. `ken` exposes two interfaces, a [command-line interface](../node/endpoint-node/ken-cli-commands.md) and the [JSON-RPC APIs](../dapp/json-rpc/README.md). `ken` runs on Linux and MacOS.
+* You [Endpoint Node](../node/endpoint-node/) is needed to connect to the Klaytn network and to issue an API call or send a transaction.
+* `ken` is a Klaytn Endpoint Node binary. `ken` exposes two interfaces, a [command-line interface](../node/endpoint-node/ken-cli-commands.md) and the [JSON-RPC APIs](../dapp/json-rpc/). `ken` runs on Linux and MacOS.
 * `ken` CLI comes with several utility and node management functions.
 
 **Smart Contract Development**
 
-* [Klaytn IDE](https://ide.klaytn.foundation/) - A browser-based compiler and IDE for the Klaytn smart contracts. 
+* [Remix Online IDE](https://remix.ethereum.org/) - A browser-based compiler and IDE with Klaytn Plugin.
```
Comment (kjeom) [inline docs/getting-started/development-environment.md:16]: https://ide.klaytn.foundation is now redirected to `remix klaytn plugin site`.
I think the `https://ide.klaytn.foundation` would be better for the users not to find the klaytn plugin of remix.

File: docs/smart-contract/deploy-guide.md [hunk 4]
```diff
@@ -2,9 +2,9 @@
 
 There are various ways of deploying a smart contract on Klaytn. This document provides a step-by-step guide to deploy a sample contract using various tools. We assume that you have a Klaytn account with enough KLAY to pay the transaction fee. To create an account, please visit [Klaytn Wallet](../toolkit/klaytn-wallet.md).
 
-## Klaytn IDE <a id="klaytn-ide"></a>
+## Remix Online IDE <a id="remix-ide"></a>
 
-Open your internet browser and go to https://ide.klaytn.foundation  
+Open your internet browser and go to [Remix Online IDE](https://remix.ethereum.org/).
```
Comment (kjeom) [inline docs/smart-contract/deploy-guide.md:7]: same comments https://github.com/klaytn/klaytn-docs/pull/524/files#r1058017456

File: docs/smart-contract/sample-contracts/erc-20/2-erc20.md [hunk 5]
```diff
@@ -1,28 +1,28 @@
...
-You can use Klaytn IDE or use truffle to deploy `MyERC20` smart contract.
+You can use Remix Online IDE or use Truffle to deploy `MyERC20` smart contract.
 
-## 2.1 Deploying smart contract using Klaytn IDE <a id="2-1-deploying-smart-contract-using-klaytn-ide"></a>
+## 2.1 Deploying smart contract using Remix Online IDE <a href="#2-1-deploying-smart-contract-using-klaytn-ide" id="2-1-deploying-smart-contract-using-klaytn-ide"></a>
 
-* Please visit the Klaytn IDE website and create a `MyERC20` contract. The complete source code was given at [Writing ERC-20 Smart Contract](1-erc20.md).
-* Prepare your account which will be used to deploy the contract. 
-  * If you do not have an account yet, create one at [https://baobab.wallet.klaytn.foundation/create](https://baobab.wallet.klaytn.foundation/create)
+* Please visit [Remix Online IDE](https://remix.ethereum.org/) and create a `MyERC20` contract. The complete source code was given at [Writing ERC-20 Smart Contract](1-erc20.md).
```
Comment (kjeom) [inline docs/smart-contract/sample-contracts/erc-20/2-erc20.md:7]: same comments https://github.com/klaytn/klaytn-docs/pull/524/files#r1058017456

File: docs/smart-contract/sample-contracts/erc-721/2-erc721.md [hunk 6]
```diff
@@ -1,36 +1,36 @@
...
-You can use Klaytn IDE or use truffle to deploy above `MyERC721Card` smart contract.
+You can use Remix Online IDE or use truffle to deploy above `MyERC721Card` smart contract.
 
-## 2.1 Deploying smart contract using Klaytn IDE <a id="2-1-deploying-smart-contract-using-klaytn-ide"></a>
+## 2.1 Deploying smart contract using Remix Online IDE <a id="2-1-deploying-smart-contract-using-klaytn-ide"></a>
 
-* Please visit the Klaytn IDE website and create a `MyERC721Card` contract. The complete source code was given at [Writing ERC-721 Smart Contract](1-erc721.md).
-* Prepare your account which will be used to deploy the contract. 
-  * If you do not have an account yet, create one at [https://baobab.wallet.klaytn.foundation/create](https://baobab.wallet.klaytn.foundation/create)
+* Please visit [Remix Online IDE](https://remix.ethereum.org/) and create a `MyERC721Card` contract. The complete source code is given at [Writing ERC-721 Smart Contract](1-erc721.md).
```
Comment (kjeom) [inline docs/smart-contract/sample-contracts/erc-721/2-erc721.md:7]: same comments https://github.com/klaytn/klaytn-docs/pull/524/files#r1058017456

File: docs/smart-contract/sample-contracts/klaytngreeter.md [hunk 7]
```diff
@@ -29,22 +29,19 @@ contract KlaytnGreeter is Mortal {
 }
 ```
 
-## Deploying KlaytnGreeter using Klaytn IDE <a id="deploying-klaytngreeter-using-klaytn-ide"></a>
+## Deploying KlaytnGreeter using Remix Online IDE <a href="#deploying-klaytngreeter-using-klaytn-ide" id="deploying-klaytngreeter-using-klaytn-ide"></a>
 
-* Please visit the [Klaytn IDE website](https://ide.klaytn.foundation) and create a `KlaytnGreeter` contract. The complete source code was given in the above.
-* Prepare your account which will be used to deploy the contract. 
-  * If you do not have an account yet, create one at [https://baobab.wallet.klaytn.foundation/create](https://baobab.wallet.klaytn.foundation/create)
+* Please visit [Remix Online IDE](https://remix.ethereum.org/) and create a `KlaytnGreeter` contract. The complete source code was given in the above.
```
Comment (kjeom) [inline docs/smart-contract/sample-contracts/klaytngreeter.md:34]: same comments https://github.com/klaytn/klaytn-docs/pull/524/files#r1058017456

File: docs/getting-started/development-environment.md (hunk 3 above)
Comment (jiwon-lieb) [inline docs/getting-started/development-environment.md:16]:
```suggestion
* [Klaytn Plugin for Remix](https://ide.klaytn.foundation) - Klaytn Plugin for Remix, a browser-based compiler and IDE.
```

File: docs/smart-contract/deploy-guide.md (hunk 4 above)
Comment (jiwon-lieb) [inline docs/smart-contract/deploy-guide.md:7]:
```suggestion
Open your internet browser and go to [Klaytn Plugin for Remix](https://ide.klaytn.foundation).
```

File: docs/smart-contract/sample-contracts/erc-20/2-erc20.md (hunk 5 above)
Comment (jiwon-lieb) [inline docs/smart-contract/sample-contracts/erc-20/2-erc20.md:7]:
```suggestion
* Please visit [Klaytn Plugin for Remix](https://ide.klaytn.foundation) and create a `MyERC20` contract. The complete source code was given at [Writing ERC-20 Smart Contract](1-erc20.md).
```

File: docs/smart-contract/sample-contracts/erc-721/2-erc721.md (hunk 6 above)
Comment (jiwon-lieb) [inline docs/smart-contract/sample-contracts/erc-721/2-erc721.md:7]:
```suggestion
* Please visit [Klaytn Plugin for Remix](https://ide.klaytn.foundation) and create a `MyERC721Card` contract. The complete source code is given at [Writing ERC-721 Smart Contract](1-erc721.md).
```

File: docs/smart-contract/sample-contracts/klaytngreeter.md (hunk 7 above)
Comment (jiwon-lieb) [inline docs/smart-contract/sample-contracts/klaytngreeter.md:34]:
```suggestion
* Please visit [Klaytn Plugin for Remix](https://ide.klaytn.foundation) and create a `KlaytnGreeter` contract. The complete source code was given in the above.
```

**Review by praveen-klaytn — APPROVED** [review]

**Comment by jiwon-lieb** [comment]
@aidan-kwon @kjhman21 이거 승인 좀 해주세요 ㅠ.ㅠ

**Review by kjeom — APPROVED** [review]

**Pull request merged**